easybuild/easyconfigs/y/YAXT/YAXT-0.6.2-gompi-2020a.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'YAXT'
+version = '0.6.2'
```

Comment:
@jedokaplan Why not update to the latest version (0.9.0), see https://www.dkrz.de/redmine/projects/yaxt/wiki/Downloads?

Reply:
@jedokaplan ping question above?